### PR TITLE
Onboarding with styles

### PR DESCRIPTION
**Pull request by koulopoulos**
* Added bullet points to 'how it works' onboarding screen.
* Added register button styling on final onboarding screen
* Changed register button route from `signInFlow` to `login`

### CODE REVIEW

**Review by emaela — CHANGES_REQUESTED** [review]
I have a small phone and so the font is a bit too big. Ideally the font size would be responsive to the screen size, but for now I think it would be sufficient to just make the font a little smaller, like 18px to match the Figma wireframes. I'll make a separate ticket for responsive font size.

IMG_3923

File: src/screens/Onboarding/OnboardingScreen.tsx [hunk 1]
```diff
@@ -1,48 +1,52 @@
...
+    };
+
     return (
-        <View style={styles.container}>
-            <Image style={styles.image} source={{uri: screenImage}}/>
-            <Text style={styles.header}> {screenHeader} </Text>
-            <Text style={styles.content}> {screenContent} </Text>
-            <Button title="Next" onPress={() => navigation.navigate(route)}/>
-        </View>
+        <TouchableHighlight style={styles.touchable}>
```
[the remark this answers is not in the document]
Comment (koulopoulos) [reply src/screens/Onboarding/OnboardingScreen.tsx:19]: This was a remnant from before the Swiper was implemented. I'll remove it in the next commit.

**Comment by koulopoulos** [comment]
The latest commit should have fixed the issue with the font.

Additionally, I spent some time cleaning up the onboarding code. One **major change** I made was to remove the `FinalOnboardingScreen.tsx` and add an optional navigation button (register button) to `OnboardingScreen.tsx`. This avoids unnecessary code duplication between `OnboardingScreen.tsx` and `FinalOnboardingScreen.tsx`, and simplifies the code base.

**Review by emaela — APPROVED** [review]
Looks good, thanks for cleaning it up!

**Pull request merged**